src/components/new-request/search.vue

```diff
@@ -97,133 +97,141 @@
+              :menu-props="{ bottom: true, offsetY: true}"
+              :value="getDesignation"
+              @input="setDesignation($event)"
+              class="mb-n3"
+              @change="setClearErrors()"
+            />
+          </v-col>
+
+          <!-- Corporate Number checkbox, only for Canadian MRAS jurisdictions -->
+          <v-col v-if="isCanadian && isMrasJurisdiction" class="d-flex justify-end">
+            <v-tooltip
+              top min-width="390"
+              content-class="top-tooltip"
+              transition="fade-transition"
+              :disabled="isMobile"
+            >
+              <template v-slot:activator="{ on }">
+                <v-checkbox
+                  v-model="noCorpNum"
+                  id="corp-num-checkbox"
+                  class="copy-small mt-0 pt-0"
+                  hide-details
+                  v-slot:label
+                  v-on="on"
+                >
+                  <template>
+                    <span v-on="on" class="copy-small">I don't have a corporate number</span>
+                  </template>
+                </v-checkbox>
+              </template>
+
+              <p>If you don't have or don't know the corporation number of the business, enter the full legal name
+                of the business in its home jurisdiction.</p>
+
+              <p>Note: If the home jurisdiction requires a name reservation, you may want to complete a name search
+                in the home jurisdiction first to ensure that the name is available and then return to BC.</p>
+            </v-tooltip>
+          </v-col>
+        </template>
+
+        <!-- Numbered company bullets -->
+        <v-col v-if="selectedCompanyType === CompanyType.NUMBERED_COMPANY" cols="12">
+          <ul class="bullet-points">
+            <li>Your business name will be the Incorporation Number assigned by the Registry.</li>
+            <li>You can change your business name at a later date.</li>
+            <li>It is not possible to request a specific Incorporation Number.</li>
+          </ul>
         </v-col>
 
-        <!-- Corporate Number checkbox, only for XPro Canadian locations -->
-        <v-col v-if="isCanadian" class="d-flex justify-end">
-          <v-tooltip
-            top min-width="390"
-            content-class="top-tooltip"
-            transition="fade-transition"
-            :disabled="isMobile"
+        <!-- Go to COLIN / Incorporate Now buttons -->
+        <v-col v-if="selectedCompanyType === CompanyType.NUMBERED_COMPANY || isFederal" cols="12"
+          class="d-flex justify-center"
+        >
+          <v-btn
+            v-if="showColinButton"
+            class="px-9"
+            id="go-to-colin-button"
+            :href="colinLink"
+            target="_blank"
           >
-            <template v-slot:activator="{ on }">
-              <v-checkbox
-                :value="getHasNoCorpNum"
-                @input="setNoCorpNum($event)"
```

Comment:
For some unexplained reason, the "input" event lagged by a tick, so basically it wasn't set when we needed it. Changing this to simply using "v-model" works fine and is cleaner.